### PR TITLE
forms

### PR DESCRIPTION
**Pull request by glaliberte05**
Add Forms documentation
- Forms service purpose
- Client Configuration
- Scenarios

### CODE REVIEW

File: guide/v0/forms.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Contact us if you need custom translations for your front-facing forms and we'll provide the necessary
+configuration for you.
+
+
+## Scenarios
+
+In addition to the client configuration, scenarios are another way to use Forms to its full extent.
+Scenarios dictate whether some form fields should be made `readOnly` or `secret`. They also dictate if
+exiting values of a form field should be filled with the existing value, be made read-only, secret, editable, or
+simply be hidden. Contact us If you need to use custom scenarios and we will work out the configuration you need.
```
Comment (jmdeejay) [inline guide/v0/forms.md:83]: *if

File: guide/v0/forms.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+The `translation` config is a JSON object with language code as keys, and custom objects as values.
+Contact us if you need custom translations for your front-facing forms and we'll provide the necessary
+configuration for you.
+
+
+## Scenarios
+
+In addition to the client configuration, scenarios are another way to use Forms to its full extent.
+Scenarios dictate whether some form fields should be made `readOnly` or `secret`. They also dictate if
+exiting values of a form field should be filled with the existing value, be made read-only, secret, editable, or
```
Comment (jmdeejay) [inline guide/v0/forms.md:82]: existing*

File: guide/v0/forms.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+# Forms
+
+The PTI Forms service exposes the various forms used to collect information on users.
+The SDK tool uses this service to provide the different forms. Form customization is available through scenarios
+and client configuration. 
+
+
+## Client configuration
+
+The client configuration can be used to customize form styles, translations and more.
```
Comment (jmdeejay) [inline guide/v0/forms.md:10]: customize form's settings (styles, translations, etc...)

File: guide/v0/forms.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+```typescript
+interface ExceptionLists {
+  embargoedList: string[];
+  highRiskList: string[];
+  regulatoryIssuesList: string[];
+  usStateBlacklist: string[];
+}
+```
+
+- `embargoedList` is a list of country codes where service providing should be denied. A default list will be provided
```
Comment (jmdeejay) [inline guide/v0/forms.md:44]: If list is short provide it, else point to it in the code.

Comment (glaliberte05) [reply guide/v0/forms.md:44]: List isn't short, but since Form's repo is private I'll provide it here. Thanks!

**Pull request merged**